tests/test_environ.py

```diff
@@ -13,6 +14,8 @@ def foos():  # pragma: no cover
 app = Application('foo', 'tests.test_environ:foos')
 
 
+@pytest.mark.skipif(os.name == "nt", reason="On Windows, environment isn't"
```

Comment:
Do not skip any test for the specific operating system. try to pass the tests on all platforms or at least provide a good reason for the lack of a feature in windows.

Reply:
I edited and enabled the tests. Now it passed all of them.